src/main/java/com/rackspace/ceres/app/model/SeriesData.java

```diff
@@ -0,0 +1,10 @@
+package com.rackspace.ceres.app.model;
+
+import java.util.Map;
+
+import lombok.Data;
+
+@Data
+public class SeriesData {
+    Map<String, String> tags;
```

Comment:
Shouldn't we have the metrics in here with the tags?

Reply:
This is a wrapper class so each tag comes out as an object in the reply.